File: PHP/car.php
class Car {
    public $id = integer;
    public $license = string;
    public $driver = string;
    public $passengers = integer;

    public function __construct($license,$driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }
    public function PrintDataCar()
    {
        echo "The name of driver is {$this->$driver->$name} with the license {$this->$license} and document {$this->$driver->$document}"
    }
}

File: Python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account("","","","")
    __passenger = int

    # def __init__(self,license,driver) -> None:
    #     self.license = license
    #     self.driver = driver

    def setPassengers(self, __passenger):
        if __passenger == 4 and __passenger != '':
            return self.__passenger
        else:
            print('You must assign 4 passengers')

    def printCarData(self, license, driver, setPassengers ) -> None:
        self.license = license
        self.driver = driver
        if setPassengers != '':
            print('\nLicense: ' + license + 
                '\nDriver Name: ' + driver.name + 
                '\nPassengers: ' + str(setPassengers))
        else:
            print('You must assign 4 passengers')